pilosa.go

```diff
@@ -201,16 +215,30 @@ func (i *Index) setupField(field *gopilosa.Field) error {
 }
 
 // SetupPilosa returns a new Indexer after creating the given fields and starting importers.
-func SetupPilosa(hosts []string, indexName string, schema *gopilosa.Schema, batchsize uint) (Indexer, error) {
+func SetupPilosa(hosts []string, indexName string, schema *gopilosa.Schema, batchsize uint, options ...PilosaOption) (Indexer, error) {
```

Comment:
can you add the example from the PR template to the doc comment on this function?  

```
pdk.SetupPilosa(...,
    pdk.OptPilosaImportOptions(gopilosa.OptImportThreadCount(4)),
    pdk.OptPilosaClientOptions(gopilosa.OptClientRetries(2)))
```

Reply:
sure, updated.